charts/argo-templates/scripts/query-non-null-rows.sh

```diff
@@ -9,13 +9,13 @@ declare -r PK_COLS=$(cat ${TABLE_DIR}/primary-keys)
 
 # Build the WHERE clause of the SQL query.
 declare -a COMPARISONS=()
-declare -a PRIMARY_COLUMNS=()
+declare -a DATAREPO_COLUMNS=(datarepo_row_id)
 for c in ${PK_COLS//,/ }; do
   COMPARISONS+=("${c} IS NOT NULL")
-  PRIMARY_COLUMNS+=("${c} as datarepo_${c}")
```

Comment:
The `as` was a syntax error.

Reply:
Whoops, sorry that was from my commit.
Is "col_name as alias" not valid for BQ sql syntax?

Reply:
It's valid in the `SELECT` list (which is how it's used in the script we copied from), but not in the `EXCEPT` list (how it's used here).